src/analysis/retail/hunter/survival/modules/talents/GrenadeJuggler.tsx

```diff
@@ -0,0 +1,83 @@
+import TALENTS from 'common/TALENTS/hunter';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { CastEvent } from 'parser/core/Events';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import { formatDurationMillisMinSec } from 'common/format';
+
+/**
+ * Grenade Juggler reduces the remaining cooldown on Wildfire Bomb by 1.0 sec on auto-attack.
```

Comment:
I think this comment is outdated. Grenade Juggler reduces the CD of Wildfire Bomb by 2 seconds when Explosive Shot is cast, not 1 second on auto-attack.